src/Client.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace GoogleTranslate;
+
+use GuzzleHttp\Client as HttpClient;
+use GuzzleHttp\ClientInterface;
+use GuzzleHttp\Exception\GuzzleException;
+
+class Client implements TranslateInterface, LanguagesInterface
+{
+    /**
+     * API URI
+     */
+    const API_URI = 'https://www.googleapis.com/language/translate/v2';
+
+    /**
+     * Access key
+     *
+     * @var string
+     */
+    private $accessKey;
+
+    /**
+     * Http client
+     *
+     * @var \GuzzleHttp\ClientInterface
+     */
+    private $httpClient;
+
+    /**
+     * Client constructor.
+     * @param string $accessKey
+     * @param ClientInterface $httpClient
+     */
+    public function __construct($accessKey, ClientInterface $httpClient = null)
+    {
+        if (strlen($accessKey) !== 39) {
```

Comment:
I suggest create an static constant to hold the '39' value. It's a good practice avoid magic numbers.